pkg/aws/policies.go

```diff
@@ -853,14 +854,23 @@ func (c *awsClient) ListOperatorRoles(version string) (map[string][]Role, error)
 					}
 					isTagged = true
 					operatorRole.Version = tagValue
+				
+				case tags.ClusterID:
```

Comment:
This actually needs to be on line 826, after the ManagedPolicies tag check, that's because the cluster ID tag is in the role and not the policy